index.js

```diff
@@ -194,18 +194,26 @@ AvlTree.prototype._deleteMax = function (node, parent) {
 	return max;
 };
 
-AvlTree.prototype.getMin = function (node) {
+AvlTree.prototype.getMin = function () {
+	return this._getMin(this._root);
```

Comment:
Hi, thanks for the PR!  Sorry for the late reply.
I think these public functions should return the element, not the node.  The idea is to try not expose the nodes to the public api.  See AvlTree.prototype.deleteMax returns the element.

Reply:
You are right! I will do that tomorrow :+1:

Reply:
I done that already :)

Reply:
Putting the .element in the private function _getMin means that _getMin has two different types it can possibly return, a node or a number.  It is better to only have 1 return type for a function.  So if you put it in getMin it would be better.